packages/ui/src/common/email-or-username-or-phone-number-field.tsx

```diff
@@ -12,12 +12,16 @@ export function EmailOrUsernameOrPhoneNumberField({
   name = undefined,
   labelEmailOrUsername = 'Email address or username',
   labelPhoneNumber = 'Phone number',
+  toggleLabelEmailOrUsername = 'Use email or username',
+  toggleLabelPhoneNumber = 'Use phone',
```

Comment:
Nit: I know I started this pattern but I'm starting to think we should probably make this a required prop to reduce the likelihood of missing a translation

I'll create a follow-up!